config_example.yml

```diff
@@ -0,0 +1,46 @@
+# This is an example config for Schematic.
+# All listed values are those that are the default if a config is not used.
+# Save this as config.yml, this will be gitignored.
+# Remove any fields in the config you don't want to change
+# Change the values of any fields you do want to change
+
+
+# This describes where assets such as manifests are stored
+asset_store:
+  # This is when assets are stored in a synapse project
+  synapse:
+    # Synapse ID of the file view listing all project data assets.
+    master_fileview_id: "syn23643253"
+    # Path to the synapse config file, either absolute or relative to this file
+    config: ".synapseConfig"
+    # Base name that manifest files will be saved as
+    manifest_basename: "synapse_storage_manifest"
+    # Location where manifests will saved to
```

Comment:
I can do that

Reply:
Should manifest basename be moved as well?

Reply:
manifest_basename refers to naming that appears on synapse, so to me it make sense to keep it in synapse, but it doest necessarily need to be there in the future... it might make sense to move it to the manifest header, it'll be important that its clear the naming between that and the title.